postman-generator/src/main/scala/models/attributes/PostmanAttributes.scala

```diff
@@ -56,5 +57,6 @@ object PostmanAttributes {
     password: String
   )
 
+  implicit val config = JsonConfiguration(SnakeCase)
```

Comment:
bugfix - sometimes in the algorithm we read `ExtendedObjectReference` so it needs to be snake case as the rest of ApiBuilder